orc8r/protos/bootstrapper.proto

```diff
@@ -101,8 +101,9 @@ message GetTokenResponse {
 }
 
 message GetGatewayRegistrationInfoRequest {
+  string network_id = 1;
   // token is a nonce prepended by bootstrapper.tokenPrepend
-  string token = 1;
+  string token = 2;
```

Comment:
I think you can't really just change the numbers here for proto definitions. Add `network_id` as `2`.